aes/decryptor.go

```diff
@@ -117,7 +135,7 @@ func (c Cipher) Encrypt(value interface{}, key []byte, additionalAuthData []byte
 	default:
 		return "", fmt.Errorf("Value to encrypt has unsupported type %T", value)
 	}
-	out := gcm.Seal(nil, iv, plaintext, additionalAuthData)
+	out := gcm.Seal(nil, iv, plaintext, []byte(path))
```

Comment:
Sure, but then I have to cast to []byte elsewhere. `path` comes directly from yaml and json keys, which are of type `string`.